spec/lib/diaspora/exporter_spec.rb

```diff
@@ -10,6 +17,14 @@
       user,
       %i[generic_user_data activity status_messages_flavours work_aspect]
     )
+
+    # verifications of data generation (protect us from possible false positives in case of poor data preset)
```

Comment:
Hmm, I think I would create a `spec/spec/` folder for specs about specs ;) I didn't expect a file called `misc_spec.rb` to test the tests.

(And I don't know why GitHub marked it as outdated)